server/routerlicious/packages/routerlicious-base/src/riddler/runnerFactory.ts

```diff
@@ -47,6 +50,25 @@ export class RiddlerResources implements IResources {
 
 export class RiddlerResourcesFactory implements IResourcesFactory<RiddlerResources> {
     public async create(config: Provider): Promise<RiddlerResources> {
+        // Cache connection
+        const redisConfig = config.get("redisForTenantCache");
+        const redisOptions: Redis.RedisOptions = {
+            host: redisConfig.host,
+            port: redisConfig.port,
+            password: redisConfig.pass,
+        };
+        if (redisConfig.tls) {
+            redisOptions.tls = {
+                servername: redisConfig.host,
+            };
+        }
+        const redisParams = {
+            expireAfterSeconds: redisConfig.keyExpireAfterSeconds as number | undefined,
+        };
+        const redisClient = new Redis(redisOptions);
+
+        const cache = new RedisCache(redisClient, redisParams);
```

Comment:
The default expiration time in `RedisCache` [is 24 hours](https://github.com/microsoft/FluidFramework/blob/1fae8d00b74e220f93d0d396910b8795ecad535f/server/routerlicious/packages/services/src/redis.ts#L16). It might be worth checking if that expiration time is still compliant for the type of data we will be adding to the cache in this context. My hunch is that this is likely not an issue, but it is something that came to my mind while I was reading the code so I thought it was worth mentioning it.

If we need a different value other than 24h, we need to remember to set it through the `keyExpireAfterSeconds` setting

Reply:
I checked with @ahmedbisht  that this should not be an issue. @sharmakhushboo